src/manifests_workflow/input_manifests.py

```diff
@@ -106,30 +111,23 @@ def update(self, min_klass: Union[Type[ComponentOpenSearchMin], Type[ComponentOp
                 self.add_to_cron(release_version)
 
     def create_manifest(self, version: str, components: List = []) -> InputManifest:
-        image_map = {
-            "opensearch": "opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2",
-            "opensearch-dashboards": "opensearchstaging/ci-runner:ci-runner-centos7-opensearch-dashboards-build-v2"
-        }
-
-        data: Dict = {
-            "schema-version": "1.0",
-            "build": {
-                "name": self.name,
-                "version": version
-            },
-            "ci": {
-                "image": {
-                    "name": image_map[self.prefix]
-                }
-            },
-            "components": [],
-        }
+        templates_base_path = os.path.join(self.manifests_path(), "templates")
+        template_version_folder = version.split(".")[0] + ".x"
+        template_full_path = os.path.join(templates_base_path, self.prefix, template_version_folder, "manifest.yml")
+        if os.path.exists(template_full_path):
```

Comment:
Avoid duplicating `InputManifest.from_file` logic, this is shorter:

```python
template_path = ...
if not os.path.exists(template_path):
    template_full_path = os.path.join(teplate_base_path, ...)

manifest = InputManifest.from_file(template_path)
```

Rename `input_manifest_template` to `manifest` since it's an actual manifest.

You can probably remove/edit the hard-coded version in the manifest templates to avoid confusion, it's a template, so maybe just say 'version: replace' or something like that.

Reply:
Updated with new commits. Please help review it. Thanks.

Reply:
See https://github.com/opensearch-project/opensearch-build/pull/2358#discussion_r931498438